rhombus/scribblings/ref-syntax-class.scrbl

```diff
@@ -10,12 +10,16 @@
 
 @doc[
   defn.macro '«syntax.class $name:
+                $maybe_description
                 pattern
                 | $clause
                 | ...»',
   defn.macro '«syntax.class $name
                | $clause
                | ...»',
+  grammar maybe_description:
+    description: $body
```

Comment:
Along the same lines as the implementation, this should be `$body; ...` to allow multiple groups.